CMakeLists.txt

```diff
@@ -9,6 +9,13 @@ set(LXQT_BUILD_TOOLS_MINOR_VERSION 3)
 set(LXQT_BUILD_TOOLS_PATCH_VERSION 2)
 set(LXQT_BUILD_TOOLS_VERSION ${LXQT_BUILD_TOOLS_MAJOR_VERSION}.${LXQT_BUILD_TOOLS_MINOR_VERSION}.${LXQT_BUILD_TOOLS_PATCH_VERSION})
 
+
+# Check for needed versions
+# We need at least Qt 5.6 and glib-2.0 >= 2.50 to build all LXQt parts 
+find_package(PkgConfig REQUIRED)
+find_package(Qt5Core "5.6.1"  REQUIRED)
+pkg_check_modules(GLIB2 glib-2.0>=2.50 REQUIRED)
```

Comment:
na - it isn't about the need - but in this case it is just: Hey - in need some Qt and som glib in a minimum version. And including the module and call it the right way are two lines too :)